### ISSUE TITLE
Classifying the wrong image in federated learning duet_mnist url_image example

### ISSUE THREAD

**Issue by mohres** [OpenMined/courses#424]
## Description
In federated-learning -> duet_mnist -> MNIST_Syft_Data_Scientist notebook, the function `classify_url_image` does not really classify the received image's url, it predicts the value of `image_1` which is an example from the test dataset.

## How to Reproduce
1. Go to 'federated-learning/duet_mnist/MNIST_Syft_Data_Scientist.ipynb'
2. Scroll down to 'classify_url_image', notice that the function only shows the received image in line `plt.imshow(im, ...`, but predicts a different one in line `class_num, preds = classify_local(image_1, local_model)`

## Expected Behavior
The expected behavior is that the function downloads the given image `image_url`, displays it, and predicts the digit that it represents. 

## Screenshots
![image](https://user-images.githubusercontent.com/34924291/178724867-171b0741-fc45-44d1-b5b4-5aa57945e5e8.png)

**Comment by mohres** [comment]
Converting the PIL,Image.Image to a tensor solves the problem,

`
transform = torchvision.transforms.ToTensor()

def image_to_tensor(im: PIL.Image.Image) -> torch.Tensor:
    """Converts given PIL.Image.Image object to torch.Tensor
    """
    
    im_transformed = transform(im)
    im_transformed = im_transformed[0].clone().detach()
    
    return im_transformed
`

and the classify_url_image function looks like this
`
def classify_url_image(image_url):
    filename = os.path.basename(image_url)
    os.system(f'curl -O {image_url}')
    im = Image.open(filename)
    im = PIL.ImageOps.invert(im)
    im = im.convert('LA')
    im = im.resize((28,28), Image.ANTIALIAS)
    enhancer = ImageEnhance.Brightness(im)
    im = enhancer.enhance(3)
    # convert the image to a torch.tensor to send it to the model for prediction  
    im_tensor = image_to_tensor(im)

    print(im.size)
    fig = plt.figure()
    plt.tight_layout()
    plt.imshow(im, cmap="gray", interpolation="none")
    
    # classify local
    class_num, preds = classify_local(im_tensor, local_model)
    print(f"Prediction: {class_num}")
    print(preds)
`
